### PR TITLE
Add support for brand create

### PR DESCRIPTION
**Pull request by nukosuke**
- [x] struct
- [x] API interface
- [x] Create implementation
- [x] test

### CODE REVIEW

File: zendesk/attachment.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	ID          int64   `json:"id"`
+	FileName    string  `json:"file_name"`
+	ContentURL  string  `json:"content_url"`
+	ContentType string  `json:"content_type"`
+	Size        int64   `json:"size"`
+	Thumbnails  []Photo `json:"thumbnails"`
+	Inline      bool    `json:"inline"`
+}
+
+type Photo struct {
```
[the remark this answers is not in the document]
Comment (nukosuke) [reply zendesk/attachment.go]: fixed

**Comment by nukosuke** [comment]
Need to prepare brand fixture

**Comment by nukosuke** [comment]
@tamccall Would you review this?

File: zendesk/brand.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+// BrandAPI an interface containing all methods associated with zendesk brands
+type BrandAPI interface {
+	CreateBrand(brand Brand) (Brand, error)
+}
+
+// CreateBrand creates new brand
+// https://developer.zendesk.com/rest_api/docs/support/brands#create-brand
+func (z *Client) CreateBrand(brand Brand) (Brand, error) {
```
Comment (tamccall) [inline zendesk/brand.go]: this is fine but there are a lot of places in here where this receiver is not a pointer.

We might need to create an issue to update all of those.

Comment (nukosuke) [reply zendesk/brand.go]: Thanks. I opened new issue about this #80

**Pull request merged**